addons/web_editor/static/src/js/editor/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -218,26 +222,44 @@ export class Powerbox {
         term = term.toLowerCase().replaceAll(/\s/g, '\\s').replaceAll('\u200B', '');
         if (term.length) {
             const regex = new RegExp(term.split('').map(char => char.replace(REGEX_RESERVED_CHARS, '\\$&')).join('.*'));
-            return commands.filter(command => `${command.groupName} ${command.title}`.toLowerCase().match(regex));
+            return commands.filter(command => `${command.category} ${command.name}`.toLowerCase().match(regex));
         } else {
             return commands;
         }
     }
+    /**
+     * Keep only the last-declared category when two categories have the same
+     * name. Return the filtered list.
+     *
+     * @param {Array<{name: string, priority: number}} categories
+     * @returns {Array<{name: string, priority: number}}
+     */
+    _removeDuplicateCategories(categories) {
```

Comment:
Ok to take it out of the method. As for the shortening that's fine except it keeps the one that is declared first rather than the one that is declared last.

Reply:
Well to have it reversed you could do this:
```js
categories.reverse.().filter((category, index, categories) => categories.findIndex(c => c.name === category.name) === index);
```

Reply:
Since `reverse` modifies the array in place, make it `[...categories] instead and we're good. Nice, I wouldn't have thought of this, pretty neat :ok_hand: